docs/source/docker/getting_started/first_steps.rst

```diff
@@ -124,6 +124,14 @@ The command above does the following:
   will be 30% of the initial dataset size. You can also specify the exact 
   number of remaining images by setting **n_samples** to an integer value.
 
+You can also use the stopping condition **min_distance**. This allows you specify
```

Comment:
There's a typo here: Final -> final.

Also, I think the wording is a little rough. What about

"This allows you to specify the minimum allowed distance between two image embeddings in the output dataset. After normalizing the input embeddings to unit length, this value should be between 0 and 2."

Reply:
I like